node/src/chain_spec/neumann.rs

```diff
@@ -174,25 +187,23 @@ pub fn neumann_staging_testnet_config() -> ChainSpec {
 		"neumann",
 		ChainType::Live,
 		move || {
-			let accounts = vec![
-				// 67nmVh57G9yo7sqiGLjgNNqtUd7H2CSESTyQgp5272aMibwS
-				hex!["488ced7d199b4386081a52505962128da5a3f54f4665db3d78b6e9f9e89eea4d"].into(),
-				// 67kgfmY6zpw1PRYpj3D5RtkzVZnVvn49XHGyR4v9MEsRRyet
-				hex!["46f630b3f79c588100dc0f69845633a830e01ea09eed4f1d01314a9bf33b9c16"].into(),
-				// 6A6VuGbeUwm3J2HqLduH7VFZTvrYQs8GuqzdhopLGN2JKMAe
-				hex!["ae8b51cd0aa290645e593a4f54673ae62bab95791a137b943723bb6070533830"].into(),
-				// 699YyPF2uA83zsFnQU4GCAvZXzucvyS5rx8LS9UrL9kEv8PP
-				hex!["84a328f5f568d82ecd91861df7eae1065c1a2f1bcfec0950d4124e9363205b4a"].into(),
-				// 67D6ecyNhnAzZqgRbxr3MdGnxB9Bw8VadMhjpLAYB3wf5Pq6
-				hex!["2edf0fd8948ea642f135b314b1358c77ec6d0a4af83220b6ea18136e5ce36277"].into(),
-				// 6AMsXyV1CYc3LMTk155JTDGEzbgVPvsX9aXp7VXz9heC3iuP
-				hex!["ba44d2c00d9528c2d1fc51cef8ce8b9c3939928ecda8f404cdc46e3a2c090627"].into(),
-				// 669ocRxey7vxUJs1TTRWe31zwrpGr8B13zRfAHB6yhhfcMud
-				hex!["001fbcefa8c96f3d2e236688da5485a0af67988b78d61ea952f461255d1f4267"].into(),
-			];
-			let initial_balance: u128 = TOTAL_TOKENS / accounts.len() as u128;
-			let endowed_accounts: Vec<(AccountId, Balance)> =
-				accounts.iter().cloned().map(|k| (k, initial_balance)).collect();
+			let allocation_json = &include_bytes!("../../../distribution/neumann_vest_test_alloc.json")[..];
```

Comment:
does this have the accoubnt listed above in them?

Reply:
added